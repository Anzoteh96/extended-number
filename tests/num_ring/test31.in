645217793117125728492113132848303833636761344649885713549174109383130562067627527464447217437
-162882947661941155995105139496438040262354338107216361251421757862702380087680871295001354789
507120895934501745108963324188427376128910282626940438463527416820867321691174204184751571929
