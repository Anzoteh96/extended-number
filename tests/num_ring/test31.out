Enter the number base b:
Enter two numbers n and m:
b is 645217793117125728492113132848303833636761344649885713549174109383130562067627527464447217437
n mod b is 482334845455184572497007993351865793374407006542669352297752351520428181979946656169445862648
m mod b is 507120895934501745108963324188427376128910282626940438463527416820867321691174204184751571929
n + m is 344237948272560589113858184691989335866555944519724077212105658958164941603493332889750217140
n - m is 620431742637808555880157802011742250882258068565614627383399044082691422356399979449141508156
n * m is 300826663333497081012527585606551927473365810951803114427514848226937110114527039373896235778
n ** m is 619020013027671827851092807649086725498444438465618773813153277782118139061723724066255002457
